apps/explorer/src/pages/transaction-result/TransactionView.tsx

```diff
@@ -479,25 +459,33 @@ function TransactionView({ txdata }: { txdata: DataType }) {
                                 className={cl([
                                     styles.txcomponent,
                                     styles.txsender,
+                                    'md:ml-4',
                                 ])}
+                                data-testid="transaction-timestamp"
                             >
-                                {txdata.timestamp_ms && (
-                                    <h3
-                                        data-testid="transaction-timestamp"
-                                        className={styles.datetime}
-                                    >
-                                        {convertNumberToDate(
-                                            txdata.timestamp_ms
-                                        )}
-                                    </h3>
+                                {txnTransfer?.[0].amount && (
+                                    <section className="mb-10">
+                                        <StatAmount
+                                            amount={txnTransfer[0].amount}
+                                            date={txdata.timestamp_ms}
+                                            currency={txnTransfer[0].coinType}
+                                        />
+                                    </section>
                                 )}
-                                <SendReceiveView
-                                    sender={sendreceive.sender}
-                                    recipient={sendreceive.recipient}
-                                    amount={sendreceive.amount}
-                                    objects={sendreceive.objects.map(
-                                        (obj) => obj.objectId
+
+                                {!txnTransfer?.[0].amount &&
```

Comment:
Why is this guarded on the first amount?

Reply:
Handles instances without txn amount to render just the date component.